rosbag2_transport/CMakeLists.txt

```diff
@@ -15,6 +15,9 @@ if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
   add_compile_options(-Wall -Wextra -Wpedantic -Werror)
 endif()
 
+# TODO(piraka9011) Remove once testing is finished
```

Comment:
When is testing going to be finished? Is there an issue to track this temp change?

Reply:
Consider adding a link to an issue instead so that we can track it.

Reply:
No, I mean that the resolution is to remove it now, before we approve the PR